src/DssSpell.sol

```diff
@@ -16,146 +16,215 @@
+
+        // Authorize new ESM to execute in new END
+        DssExecLib.authorize(MCD_END, MCD_ESM);
+
+        // Authorize new ESM to execute in VAT
+        DssExecLib.authorize(MCD_VAT, MCD_ESM);
+
+        // Make every flipper relies the MCD_ESM
+        for (uint256 i = 0; i < flips.length; i++) {
+            DssExecLib.authorize(flips[i], MCD_ESM);
+        }
+
+        // ------------------  DOG  ------------------
+
+        require(DogAbstract(MCD_DOG).vat() == MCD_VAT, "DssSpell/error-dog-vat");
+
+        // Set VOW in the DOG
+        DssExecLib.setContract(MCD_DOG, "vow", MCD_VOW);
+
+        // Authorize DOG can access to VAT
+        DssExecLib.authorize(MCD_VAT, MCD_DOG);
+
+        // Authorize DOG can access to VOW
+        DssExecLib.authorize(MCD_VOW, MCD_DOG);
+
+        Fileable(MCD_DOG).file("Hole", 100 * MILLION * RAD);
+
+
+        // --------------  CLIPPER_MOM  --------------
+
+        require(ClipperMomAbstract(CLIPPER_MOM).spotter() == MCD_SPOT, "DssSpell/error-clipper-mom-spotter");
+
+        ClipperMomAbstract(CLIPPER_MOM).setAuthority(DssExecLib.getChangelogAddress("MCD_ADM"));
+
+        // ----------------  LINK-A  -----------------
+
+        require(ClipAbstract(MCD_CLIP_LINK_A).vat() == MCD_VAT, "DssSpell/error-clip-vat");
+        require(ClipAbstract(MCD_CLIP_LINK_A).dog() == MCD_DOG, "DssSpell/error-clip-dog");
+        require(ClipAbstract(MCD_CLIP_LINK_A).spotter() == MCD_SPOT, "DssSpell/error-clip-spot");
+        require(ClipAbstract(MCD_CLIP_LINK_A).ilk() == "LINK-A", "DssSpell/error-clip-ilk");
+
+        // Set CLIP for LINK-A in the DOG
+        DssExecLib.setContract(MCD_DOG, "LINK-A", "clip", MCD_CLIP_LINK_A);
+
+        // Set VOW in the LINK-A CLIP
+        DssExecLib.setContract(MCD_CLIP_LINK_A, "vow", MCD_VOW);
+
+        // Set CALC in the LINK-A CLIP
+        DssExecLib.setContract(MCD_CLIP_LINK_A, "calc", MCD_CLIP_CALC_LINK_A);
+
+        // Authorize CLIP can access to VAT
+        DssExecLib.authorize(MCD_VAT, MCD_CLIP_LINK_A);
+
+        // Authorize CLIP can access to DOG
+        DssExecLib.authorize(MCD_DOG, MCD_CLIP_LINK_A);
+
+        // Authorize DOG can kick auctions on CLIP
+        DssExecLib.authorize(MCD_CLIP_LINK_A, MCD_DOG);
+
+        // Authorize the new END to access the LINK CLIP
+        DssExecLib.authorize(MCD_CLIP_LINK_A, MCD_END);
+
+        // Authorize CLIPPERMOM can set the stopped flag in CLIP
+        DssExecLib.authorize(MCD_CLIP_LINK_A, CLIPPER_MOM);
+
+        // Authorize new ESM to execute in LINK-A Clipper
+        DssExecLib.authorize(MCD_CLIP_LINK_A, MCD_ESM);
+
+        // Whitelist CLIP in the LINK osm
+        DssExecLib.addReaderToOSMWhitelist(PIP_LINK, MCD_CLIP_LINK_A);
+
+        // Whitelist CLIPPER_MOM in the LINK osm
+        DssExecLib.addReaderToOSMWhitelist(PIP_LINK, CLIPPER_MOM);
+
+        // No more auctions kicked via the CAT:
+        DssExecLib.deauthorize(MCD_FLIP_LINK_A, MCD_CAT);
+
```

Comment:
We forgot `cat.deny(flip)` in the checklist. I just added it.

```suggestion

        // No more claws by the FLIP:
        DssExecLib.deauthorize(MCD_CAT, MCD_FLIP_LINK_A);
        
```

Reply:
This has to be removed in a subsequent spell as there may be active auctions when this spell executes.

Reply:
Yes exactly, this can't be done now. And btw the idea is leaving for a while the flip auctions fully set up just in case we need to roll back.

Reply:
Btw I've just removed it from the checklist.

Reply:
resolving